tailwind.config.ts

```diff
@@ -1,16 +1,19 @@
-import type { Config } from "tailwindcss";
+import type { Config } from 'tailwindcss';
 
 const config: Config = {
   content: [
-    "./src/pages/**/*.{js,ts,jsx,tsx,mdx}",
-    "./src/components/**/*.{js,ts,jsx,tsx,mdx}",
-    "./src/app/**/*.{js,ts,jsx,tsx,mdx}",
+    './src/pages/**/*.{js,ts,jsx,tsx,mdx}',
+    './src/components/**/*.{js,ts,jsx,tsx,mdx}',
+    './src/app/**/*.{js,ts,jsx,tsx,mdx}',
   ],
   theme: {
     extend: {
       colors: {
-        background: "var(--background)",
-        foreground: "var(--foreground)",
+        'background-dark': '#282A36',
+        'background-darker': '#20222C',
+        'primary-accent': '#9D7CCF',
+        'secondary-blue': '#6272A4',
+        'highlight-lavender': '#9D7CCF',
```

Comment:
make this a var color in css then put those in tailwind

Reply:
will do now 😄